### DISCUSSION TITLE
Button - Datenpunktwert setzen

### DISCUSSION THREAD

**Discussion by Chris0706** [Zefau/ioBroker.jarvis#1423] [Ideas / Feature Requests]
Hallo,

gibts eigentlich eine Möglichkeit einen frei definierbaren Button zu erstellen? Ich würde z.B. gerne für meine Heizungen zwei zusätzliche Buttons haben, denen fixe Temperaturen zugeordnet sind. Einer mit 17°C und einer mit 21°C. Diese müssten dann beim Drücken, den Datenpunkt hm-rpc.XXXXX.SET_POINT_TEMPERATURE eben entweder auf 17 oder 21 setzen.

Grüße
Chris

**Comment by Zefau** [comment]
Du kannst einen eigenen State im jarvis Device definieren (jeweils für 17°C und 21°C nötig). Diesen dann einen Button zuordnen und in den Datenpunkt Eigenschaften `setOn` definieren mit dem gewünschten Wert (also 17 bzw. 21). Das sollte gehen.
